src/Liquidator.sol

```diff
@@ -474,19 +445,19 @@ contract Liquidator is Owned, ILiquidator {
     }
 
     function _knockDown(address account, AuctionInformation storage auctionInformation_) internal {
+        // Set the inAuction flag to false.
+        auctionInformation[account].inAuction = false;
+
         (bool success,,) = IAccount(account).isAccountHealthy(0, 0);
```

Comment:
do we still need this with (0,0)?

Reply:
yes different PR

Reply:
https://arcadiafinance.atlassian.net/browse/AF-376